janaganana/settings.py

```diff
@@ -1,20 +1,41 @@
 # pull in the default wazimap settings
 from wazimap.settings import *  # noqa
+import os
 from decouple import config
 
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
 DEBUG = config('DJANGO_DEBUG', default=True, cast=bool)
 TEMPLATE_DEBUG = DEBUG
 
+SECRET_KEY = 'wt08^t@ugzs4sw*qn=c*=$d+jgkqkkp4$0z98j-k5s!o2um$(n'
```

Comment:
In general, and in this case in particular, the secrets are read from config file in production.Having Secretkey in settings is not recpmmend.I used decouple libarary to read it from config file.

Reply:
I have used decouple package for the secret key.